.github/workflows/close_inactive_issues.yml

```diff
@@ -1,26 +1,49 @@
 name: Close inactive issues
 on:
   schedule:
+    # Everyday at 1:30PM
     - cron: "30 1 * * *"
 
 jobs:
   close-issues:
     runs-on: ubuntu-latest
     permissions:
       issues: write
-      pull-requests: write
```

Comment:
Is this where pull-requests doesn't get send notifications?

Reply:
It's unrelated AFAIK. The gitbot would have to be configured in a really really really weird way for this to be the cause - particularly because this hasn't been merged into `development`, or `stable` yet. Also, the workflows only happen on the `stable` branch I believe (they only run on the default branch unless otherwise specified).

Reply:
That is so right. Hmmm...